sdk/src/main/scala/io/horizen/account/api/rpc/types/TxPoolContentFrom.scala

```diff
@@ -0,0 +1,12 @@
+package io.horizen.account.api.rpc.types
+
+import com.fasterxml.jackson.annotation.JsonView
+import io.horizen.json.Views
+
+import java.math.BigInteger
+import scala.collection.mutable
+
+@JsonView(Array(classOf[Views.Default]))
+class TxPoolContentFrom(
+    val pending: mutable.SortedMap[BigInteger, TxPoolTransaction],
```

Comment:
maybe here as well: nonce is decimal.
Note: I can't find in the docs proper example.

Reply:
Same as `txpool_content` as reported in the comment above